hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/Networking.java

```diff
@@ -129,23 +131,26 @@ private void broadcastFlowControlPacket() {
         }
     }
 
-    private byte[] createFlowControlPacket(Address member) throws IOException {
+    private byte[] createFlowControlPacket(Address member, Connection expectedConnection) throws IOException {
         try (BufferObjectDataOutput output = createObjectDataOutput(nodeEngine, lastFlowPacketSize)) {
-            final boolean[] hasData = {false};
+            boolean hasData = false;
             Map<Long, ExecutionContext> executionContexts = jobExecutionService.getExecutionContextsFor(member);
             output.writeInt(executionContexts.size());
-            executionContexts.forEach((execId, exeCtx) -> uncheckRun(() -> {
```

Comment:
Is this change mostly about "FP delete"? The new code has explicit types on everything, but with the types being Integer, Map etc., their true meaning is not known, and previously there were lambda parameters with descriptive names. I don't see a win here.

Reply:
I guess FP means functional programming. Then yes. It also creates less garbage and it's easier to debug.

The only real change is the argument to `updateAndGetSendSeqLimitCompressed` IIRC.